exercises/resistor-color-duo/.meta/hints.md

```diff
@@ -8,3 +8,32 @@ value :: (Color, Color) -> Int
 
 that only accepts two colors and produces the resistor's numeric value
 component of its resistance.
+
+**Bonus - Property Testing:**
+
+This exercise is a good opportunity to write some [property tests](https://en.wikipedia.org/wiki/Property_testing). If you haven't written any before here are some resources:
+
+1. [School of Haskell](https://www.schoolofhaskell.com/user/pbv/an-introduction-to-quickcheck-testing).
+2. [Real world Haskell](http://book.realworldhaskell.org/read/testing-and-quality-assurance.html)
+
+We have provided one sample property test in the [test file](test/Tests.hs). We use the [QuickCheck](https://hackage.haskell.org/package/QuickCheck) library, but there are [others](http://hackage.haskell.org/package/hedgehog) as well, feel free to play around.
+
+Few properties that can be tested:
+
+* `value (x, y) >= 10` for all x except Black and all y
+
+* `show (value (x, y)) == reverse (show (value (y, x)))` for all x, y except Black.
+
+* (Color, Color) is isomorphic to [0..99] (or: all colors have a unique value).
+
+* Associativity: Kind of tricky because you can't get the value of a single Color:
+
+```haskell
+value (k, x) <= value (k, y) && value (k, y) <= value (k, z)
+  ⇒ value (k, x) <= value (k, z)
+
+value (x, k) <= value (y, k) && value (y, k) <= value (z, k)
+  ⇒ value (k, x) <= value (k, z)
+```
```

Comment:
So, these are copied from #867 and were mainly intended for contributors to base their phrases on. I had imagined something that doesn't just involve copy-pasting the README into the test file. For the first property, that might be okay, but for subsequent properties, how about something like...

I seem to recall that canonical descriptions use `-` for unordered lists, since `*` is also bold.

I never liked the phrasing of the first property in this list, so rephrasing it below...

```suggestion
- `value (x, y) >= 10` for all x, y where x isn't Black.

- Calling `show` on `value (x, y)` produces the `reverse` String of calling `show` on `value (y, x)`.

- All colors have a unique value.

- ...
```